src/mito2/src/cache/write_cache.rs

```diff
@@ -96,3 +163,88 @@ pub struct SstUploadRequest {
     /// Remote object store to upload.
     pub remote_store: ObjectStore,
 }
+
+#[cfg(test)]
+mod tests {
+    use api::v1::OpType;
+    use common_base::readable_size::ReadableSize;
+    use common_test_util::temp_dir::create_temp_dir;
+    use object_store::manager::ObjectStoreManager;
+    use object_store::services::Fs;
+    use object_store::ObjectStore;
+    use store_api::storage::RegionId;
+
+    use super::*;
+    use crate::cache::file_cache::{self, FileCache};
+    use crate::cache::test_util::new_fs_store;
+    use crate::sst::file::FileId;
+    use crate::sst::location::sst_file_path;
+    use crate::test_util::sst_util::{
+        new_batch_by_range, new_source, sst_file_handle, sst_region_metadata,
+    };
+    use crate::test_util::{build_rows, new_batch_builder, CreateRequestBuilder, TestEnv};
+
+    #[tokio::test]
+    async fn test_write_and_upload_sst() {
+        // TODO(QuenKar): maybe find a way to create some object server for testing,
```

Comment:
We already configured S3 in github actions, please refer to https://github.com/GreptimeTeam/greptimedb/blob/1fc168bf6aee8e9a7c0c47c0461fe27745d24349/tests-integration/src/test_util.rs#L89

Reply:
yes, I also find that. however, at present, we just need to test the function of writing and uploading Sst in unit test. There is no need to put it in integration. 
what I want is a slight mock S3 server that can build in a unit test, but I doesn't find solution now, so just use local file system to mock the same behavior. 
Theoretically, if it succeed in the local file system, it should also be successful for other object stores, because this function doesn't care what the storage behind `remote_store`.

Reply:
After the `write cache` process runs through, we can benchmark for different remote stores. Here I want to confirm the correctness of the function behavior.